Themes/default/Themes.template.php

```diff
@@ -476,34 +465,33 @@ function template_set_settings()
 					<a href="', $scripturl, '?action=helpadmin;help=theme_settings" onclick="return reqOverlayDiv(this.href);" class="help"><span class="generic_icons help" title="', $txt['help'], '"></span></a> ', $txt['theme_settings'], ' - ', $context['theme_settings']['name'], '
 				</h3>
 			</div>
-			<br>';
+			<div class="windowbg">';
 
 	// @todo Why can't I edit the default theme popup.
 	if ($context['theme_settings']['theme_id'] != 1)
 		echo '
-			<div class="cat_bar">
-				<h3 class="catbg config_hd">
-					', $txt['theme_edit'], '
-				</h3>
-			</div>
-			<div class="windowbg">
-				<ul>
-					<li>
-						<a href="', $scripturl, '?action=admin;area=theme;th=', $context['theme_settings']['theme_id'], ';', $context['session_var'], '=', $context['session_id'], ';sa=edit;filename=index.template.php">', $txt['theme_edit_index'], '</a>
-					</li>
-					<li>
-						<a href="', $scripturl, '?action=admin;area=theme;th=', $context['theme_settings']['theme_id'], ';', $context['session_var'], '=', $context['session_id'], ';sa=edit;directory=css">', $txt['theme_edit_style'], '</a>
-					</li>
-				</ul>
-			</div>';
+				<div class="title_bar">
+					<h3 class="titlebg config_hd">
```

Comment:
config_hd is a big icon and its not really fitting at title_bar/titlebg, maybe its best to remove the icons.
cc: @Gwenwyfar

Reply:
Not relevant to the PR IMO, just leave it be for now.